db/c_test.c

```diff
@@ -2956,6 +2956,51 @@ int main(int argc, char** argv) {
     CheckNoError(err);
   }
 
+  StartPhase("filter_with_prefix_seek");
+  {
+    rocksdb_close(db);
+    rocksdb_destroy_db(options, dbname, &err);
+    CheckNoError(err);
+
+    rocksdb_options_set_prefix_extractor(
+        options, rocksdb_slicetransform_create_fixed_prefix(1));
+    rocksdb_filterpolicy_t* filter_policy =
+        rocksdb_filterpolicy_create_bloom_full(8.0);
+    rocksdb_block_based_options_set_filter_policy(table_options, filter_policy);
```

Comment:
Are these necessary for the test?  Seems like the slice transform is what is necessary, not the filter policy

Reply:
They are both necessary. We use prefix extractor to extract the prefix and look up in the bloom filter. Before this PR, the bloom filter will return "the key does not exist".